libs/ledger-live-common/src/hw/connectApp.ts

```diff
@@ -147,6 +150,10 @@ export const openAppFromDashboard = (
                   case 0x6985:
                   case 0x5501:
                     return throwError(new UserRefusedOnDevice());
+                  // case 0x5515:
```

Comment:
What?, why?

Reply:
oh i was testing if catching this error here (`openAppFromDashboard`) was necessary or not. If we don't catch it here, it is thrown again, and it's the `cmd` (defined below) who catches it. 

I tested without catching it here, and indeed it is catched below by the `cmd`.

It looks like `openAppFromDashboard` is only used in this file, by `cmd`. But it is still exported. So I guess, just to be bulletproof, i should uncomment this catch ?

(and this commit is still WIP ;) )